git_wrapper/clone.py

```diff
@@ -84,7 +84,7 @@ def destroy_and_reclone(self):
         for name, url in remotes.items():
             if name != default_remote:
                 try:
-                    logger.debug("Adding remote {remote}".format(remote=name))
+                    logger.debug("Adding remote %s", name)
```

Comment:
Not sure, why we're reverting to old string formatting?

Reply:
Note the use of a comma instead of the % operator. This defers string formatting until it's actually used, which is useful for debug/info level messages that may not be parsed at all.